cmd/run.go

```diff
@@ -145,6 +145,7 @@ the data document with the following syntax:
 	runCommand.Flags().StringVarP(&params.OutputFormat, "format", "f", "pretty", "set shell output format, i.e, pretty, json")
 	runCommand.Flags().BoolVarP(&params.Watch, "watch", "w", false, "watch command line files for changes")
 	setMaxErrors(runCommand.Flags(), &params.ErrorLimit)
+	runCommand.Flags().BoolVarP(&params.PprofEnabled, "pprof-enabled", "", true, "enables pprof endpoints")
```

Comment:
Thinking about this is a bit more, I'd prefer to keep this off by default (for now). If we get feedback from the community indicating it could be on-by-default then we can change it later. I'd rename the flag to just `pprof` and make the default `false`.

Reply:
Ok. I'll change it

Reply:
Cool, thanks. It's just easier to go from off-by-default to on-by-default than the reverse in the future.